### ISSUE TITLE
Screw or scroll expander

### ISSUE THREAD

**Issue by David-Mamrilla** [ibell/pdsim#72]
Hello,
I have installed PDSim and am very impressed with the simulation interface for scroll and piston expanders. I would like to kindly ask, does anyone posses models for a scroll or screw expander? I saw that there is a model for a piston expander in the "examples" folder, but I was wondering whether it is possible to implement it with the GUI interface?

Thank you.

**Comment by ibell** [comment]
Did you check out the models from Paper II?

Yes, it could be implemented, if you wanted to do the work to do so, or if
you had funding for me to do it.

**Comment by David-Mamrilla** [comment]
Hello, thank you for replying,

Yes, I did check both of the papers. and read carefully especially the part about screw expanders in paper II. However, would it possible to find and use the actual python model? My intent would be to use it for academic purposes only, for my Master thesis on possibilities of PD expanders in steam cycles.

Thank you.

**Comment by ibell** [comment]
The files are here, but seem to be embargoed:
https://figshare.com/articles/code/PDSim_source/7985651.  I'll investigate.

**Comment by David-Mamrilla** [comment]
Thank you :)

**Comment by ibell** [comment]
Embargo lifted (should have been lifted a long time ago).

**Comment by David-Mamrilla** [comment]
Thank you for unlocking the files. Maybe I am not looking in the right folders but I cannot see a screw expander model? It seems that the files on figshare are the same ones as here on Github, with examples of scroll and reciprocating compressors and a piston expander, but no screw.
Thank you and sorry if it is in one of the folders and I just didn't see it.

**Comment by ibell** [comment]
@davideziviani Any idea where the examples went to? I agree they don't seem to be in the figshare repo from paper I

**Comment by David-Mamrilla** [comment]
Hello, 
have you by any chance managed to find the examples?

**Comment by MAKlNE** [comment]
Hello,
I can only use Recip compressor and scroll compressor in PDSim app.

Is the Scroll Expander locked?

Kind Regards.
Alan

**Comment by David-Mamrilla** [comment]
No, it is not locked. Only recip and scroll compressor have been implemented in the app. However, there should be python models for other machines, among them the scroll and screw expanders. But it seems the authors can't find them at the moment.
